### PR TITLE
Client error display #123

### PR DESCRIPTION
**Pull request by nicknapoli82**
Created a component called ErrorList which displays any error that we create in the redux store slice errorReducer. 
- Errors can be created
- Errors can be removed

errorReducer uses a (context, string) system. If the context does not exist it will make it, and append the errors as strings to that context. Its up to us, as to how what would get resolved to remove the error. Some may be just fine as a timed display.

Basically now, everywhere we have console.log(e) on the client side, we can produce a meaningful error for the user should we want.

See my noted in the errorReducer.js file

### CODE REVIEW

**Review by roboMoose — APPROVED** [review]
Nice, this will be really helpful for the UI and error handling.
